resources/prometheus/downstream/grafana-dashboard-dataplane-operator.yaml

```diff
@@ -0,0 +1,1441 @@
+---
+annotations:
```

Comment:
Add configmap content. Check How other dashboard file are written

Reply:
Added the configmap content